diagnostic.md

```diff
@@ -55,10 +70,13 @@ const ask = require('../lib/ask.js');
 let answer = 'not empty';
 
 while (answer !== '') {
-  answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
-   console.log("You guessed it!");
+  answer = ask("Name a number between 1 and 10:  ");
+  if (answer >= 1 && answer <= 10) {
```

Comment:
Ahh creative. I'll have to reword the question to make it a guess a number next time 👍